packages/php-domain/src/DependencyInjection/PhpDomainExtension.php

```diff
@@ -5,30 +5,41 @@
 namespace T3Docs\PhpDomain\DependencyInjection;
 
 use function dirname;
+use function phpDocumentor\Guides\DependencyInjection\template;
 
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Extension\Extension;
 
+use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
 
+use T3Docs\PhpDomain\Nodes\FullyQualifiedNameNode;
+use T3Docs\PhpDomain\Nodes\PhpComponentNode;
+use T3Docs\PhpDomain\Nodes\PhpNamespaceNode;
+
 final class PhpDomainExtension extends Extension implements PrependExtensionInterface
 {
     /** @param mixed[] $configs */
     public function load(array $configs, ContainerBuilder $container): void
     {
+        $container->prependExtensionConfig(
```

Comment:
You need to put these lines in 'prepend(ContainerBuilder $container): void {}' otherwise it will not work. 

The load method cannot extend the configuration.